app/controllers/application_controller.rb

```diff
@@ -4,7 +4,7 @@ class ApplicationController < ActionController::API
 
   # temporary current_user testing stub until we add in authentication
   def current_user
-    @current_user ||= User.find_by(email: "test@test.com")
+    @current_user ||= self.authenticate_user
   end
 
```

Comment:
This is where the hang-up was.  I had RSpec stubbed here and forgot to refactor for the actual JWT method that's required.